.github/workflows/windows-macos.yml

```
@@ -147,9 +147,12 @@ jobs:
     - name: Install dependencies from brew
       if: ${{ runner.os == 'macOS' }}
       run: |
+        # Avoid reinstalling cmake
+        ${{ env.ARCH_WRAPPER }} \
+        brew list cmake > /dev/null || ${{ env.ARCH_WRAPPER }} brew install cmake
```

This line split is a little difficult to spot. I would prefer splitting after the or and indent the second line:

```suggestion
        ${{ env.ARCH_WRAPPER }} brew list cmake > /dev/null || \
          ${{ env.ARCH_WRAPPER }} brew install cmake
```

WDYT?